src/Main.java

```diff
@@ -19,16 +19,22 @@ public static void main(String[] args) {
                     String inputPassword = scanner.nextLine();
                     LoginCommand loginCommand = new LoginCommand();
                     loginCommand.execute(inputName, inputPassword);
+
```

Comment:
the same style everywhere

Reply:
where ?

Reply:
Each line before `break` is not empty:
line 37
line 42
line 47

Reply:
Each line before break is not empty:
line 37
line 42
line 47